src/lecturer/components/reactionReceiveView/ReactionCounter.tsx

```diff
@@ -30,7 +30,8 @@ export function ReactionCounter(props: ReactionCounterProps) {
         setTimeout(() => {
             setChanged(true);
         },150)
-    }, [store.reactionValues[props.index]]);
```

Comment:
The animation now plays for every reaction. Before it was only for the one that changed :/
![image](https://user-images.githubusercontent.com/26445545/118396088-16228f00-b64e-11eb-8ff9-eec06eade635.png)